lib/safer_rails_console/patches/sandbox/auto_rollback.rb

```diff
@@ -12,7 +12,7 @@ def self.rollback_and_begin_new_transaction
 
         def self.handle_and_reraise_exception(e)
           if e.message.include?('PG::ReadOnlySqlTransaction')
-            puts color_text('An operation could not be completed due to read-only mode.', RED) # rubocop:disable Rails/Output
+            puts color_text('An operation could not be completed due to read-only mode.', Colors::RED) # rubocop:disable Rails/Output
```

Comment:
I'd use the entire namespace from the root to be safe - `::SaferRailsConsole::Colors::RED`. The fact that `RED` resolved correctly was serendipitous.

A better way to do this would be to `include SaferRailsConsole::Colors` instead of `extend ...`, ensuring that the constants are included into the metaclass. That'd involve a little bit of shuffling within `Colors` though, so no pressure and the first solution is totally adequate!

Reply:
I thought about including, but then the `color_text` won't be available.

Reply:
Yup, the shuffling I mentioned would involve putting that into a `ClassMethods` or similar module and using the popular `self.included; extend ClassMethods` pattern.